lib/pidSupport.py

```diff
@@ -204,7 +194,7 @@ def parse_pid_file_content(self, lines):
 
         try:
             pid = int(pidarr[1])
-        except:
+        except Exception:
             raise RuntimeError("Corrupted lock file: invalid PID")
```

Comment:
Should we add `from None` here for consistency?
```suggestion
        try:
            pid = int(pidarr[1])
        except Exception:
            raise RuntimeError("Corrupted lock file: invalid PID") from None
```

Reply:
done